console.rb

```diff
@@ -12,6 +12,80 @@
 # puts "The first apartment is at #{Apartment.first.address}."
 # puts "It has  #{Apartment.first.tenants.count} tenants."
 
-binding.pry
+# binding.pry
 
-puts "ignore this line, it's just here so the binding.pry above works"
+choice = nil
+while choice != 'q'
+
+  2.times {puts "*" * 70}
+  puts "Welcome to Landlord
+        Options:
+        1) See all apartments
+        2) See all tenants
+        3) See all apartments and associated tenants
+        4) Look up apartment from tenant name
+        q) Exit the program"
+  2.times {puts "*" * 70}
+  puts ""
+  puts "Please enter your selection"
+
+  choice = nil
+  while ![1, 2, 3, 4, 'q'].include? choice
+    choice = (gets.chomp)
+    choice = choice.to_i unless choice == 'q'
+  end
+
+  case choice
+  when 1
+    puts 'All apartments'
+    puts 'ID    | Address                  | Rent   | Sq Feet  | Beds | Baths'
+    puts '*' * 70
+    all_apartments = Apartment.all
+    all_apartments.each do |apartment|
+      print apartment.id, (' ' * (6-apartment.id.to_s.length)), '| ',
```

Comment:
Clever approach to alignment! It sometimes makes sense to put this sort of code in an instance method in the Apartment class for reusability.